packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -55,7 +56,19 @@ export function run() {
       log.debug('Loading account from JUPITERONE_ACCOUNT environment variable');
       const account = getAccountFromEnvironment();
 
-      const apiBaseUrl = getApiBaseUrl({ dev: options.development });
+      let apiBaseUrl: string;
```

Comment:
This overall logic is used in all three places with the only difference being what `development` mechanism is checked and what error is thrown.  We might look to try and standardize into a helper function when we come back through to add this functionality to additional commands.